.github/actions/deploy-tinybird/action.yml

```
@@ -0,0 +1,29 @@
+      uses: actions/checkout@v4
+    - name: Install Tinybird CLI
+      shell: bash
+      run: curl -fsSL https://tinybird.co/install.sh | sh
+    - name: Check the deployment
+      shell: bash
+      run: tb --cloud --host ${{ inputs.host }} --token ${{ inputs.token }} deployment create --check
+      working-directory: ghost/core/core/server/data/tinybird
+    - name: Create a ${{ inputs.workspace }} deployment
+      shell: bash
+      run: tb --cloud --host ${{ inputs.host }} --token ${{ inputs.token }} deployment create --wait
+      working-directory: ghost/core/core/server/data/tinybird
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)